packages/@ourworldindata/types/src/siteTypes/MultiDimDataPage.ts

```diff
@@ -6,9 +6,27 @@ import {
     OwidVariableWithSource,
 } from "../OwidVariable.js"
 
-// Indicator ID, catalog path, or maybe an array of those
-export type IndicatorEntryBeforePreProcessing = string | number
-export type IndicatorEntryAfterPreProcessing = number // catalog paths have been resolved to indicator IDs
+export type IndicatorConfig = Pick<OwidVariableWithSource, "id" | "display">
+
+type IndicatorConfigBeforePreProcessing = Omit<IndicatorConfig, "id"> & {
+    // Indicator ID or catalog path
+    id: number | string
```

Comment:
I'm wondering if now, we should be using `catalogPath` here specifically when referring to these variables by path?
In the past, I opted to have them as a union type precisely because it was a flat property and not an object. But now that we have this as an object either way, this all doesn't apply any more, and maybe the logic gets easier if we don't have to have so many type-based conditionals on it?

Reply:
Good point, changed.